server/client_rest.go

```diff
@@ -95,8 +95,9 @@ func (client *clientRest) FindProposals(nodeKey string) (proposals []dto_discove
 	return
 }
 
-func (client *clientRest) SessionSendStats(sessionId string, sessionStats dto.SessionStats) (err error) {
-	response, err := client.doPostRequest("client_send_stats", sessionStats)
+func (client *clientRest) CreateSessionStats(sessionId string, sessionStats dto.SessionStats) (err error) {
+	path := fmt.Sprintf("sessions/%s/stats", sessionId)
+	response, err := client.doPostRequest(path, sessionStats)
```

Comment:
function name misleading - this doesn't create but does send the stats

Reply:
Agreed.

Reply:
Done! 🍭